### PR TITLE
feat(lib): remove version check of resty.events

### PR DESCRIPTION
**Pull request by bungle**
### Summary

lua-resty-events is still in active development, we should support more possible versions.

KAG-4760

### CODE REVIEW

**Review by gszr — APPROVED** [review]

**Review by gszr — DISMISSED** [review]

**Review by gszr — APPROVED** [review]

**Review by gszr — APPROVED** [review]

**Review by gszr — APPROVED** [review]

File: .github/workflows/build_and_test_with_resty_events.yml [hunk 1]
```diff
@@ -75,14 +73,12 @@ jobs:
 
       - name: Checkout lua-resty-events
         uses: actions/checkout@v3
-        if: steps.cache-deps.outputs.cache-hit != 'true'
         with:
           repository: Kong/lua-resty-events
           ref: refs/tags/0.2.1
```
Comment (locao) [inline .github/workflows/build_and_test_with_resty_events.yml]: Should we bump this tag?

**Review by locao — APPROVED** [review]
The changes LGTM, but wouldn't it be better if we improved the cache key instead of removing the cache at all?

**Comment by bungle** [comment]
> The changes LGTM, but wouldn't it be better if we improved the cache key instead of removing the cache at all?

Perhaps with time. My conclusion is that we have lost more time to caching issues that it has ever given time back.

**Review by gszr — APPROVED** [review]

File: .github/workflows/build_and_test_with_resty_events.yml (hunk 1 above)
Comment (bungle) [reply .github/workflows/build_and_test_with_resty_events.yml]: bumped

**Pull request merged**